analysis/include/HistoManager.h

```diff
@@ -77,7 +78,7 @@ class HistoManager {
         void Fill1DHisto(const std::string& histoName, float value, float weight=1.);
         void Fill2DHisto(const std::string& histoName, float valuex, float valuey, float weight=1.);
 
-        virtual void GetHistosFromFile(TFile* inFile, const std::string& name,const std::string& folder = "");
+        virtual void GetHistosFromFile(TFile* inFile, const std::vector<std::string>& name,const std::string& folder = "");
```

Comment:
This is potentially dangerous since it modifies an already existing method. Would be safer to just overload the method with the vector version and have both available.